disable_autologin.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: contestants
+  tasks:
+    - name: disable autologin to user p
```

Comment:
I think if you remove the file here it would have the same impact:
Commenting something out or removing has the same effect
Without the AutomaticLogin lines the files doesn't list anything new so is considered empty

So I would try to remove the file here (with the stat module)

This could be personal preference though.

Reply:
by default that file exists.  Which is why i did not want to delete it.

Reply:
Than this is probably the best option.